pkg/get/tools.go

```diff
@@ -1218,10 +1218,11 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 
 	tools = append(tools,
 		Tool{
-			Owner:   "argoproj-labs",
-			Repo:    "argocd-autopilot",
-			Name:    "argocd-autopilot",
-			Version: "0.2.1",
+			Owner:       "argoproj-labs",
+			Repo:        "argocd-autopilot",
+			Name:        "argocd-autopilot",
+			Description: "The Argo-CD Autopilot is a tool which offers an opinionated way of installing Argo-CD and managing GitOps repositories.",
```

Comment:
Description is a bit verbose, why don't we just make it: 

```
An opinionated way of installing Argo-CD and managing GitOps repositories.
```

Reply:
Fine for me 😄